test/unit/atomic_iteration_test.lua

```diff
@@ -133,7 +133,6 @@ end
 -- it's not check tarantool or vinyl as engine
```

Comment:
typo in commit message "therefor" -> "therefore"

Reply:
Fixed.